airflow/www/static/js/dag/details/taskInstance/Logs/LogBlock.tsx

```diff
@@ -59,10 +67,43 @@ const LogBlock = ({ parsedLogs, wrap, tryNumber }: Props) => {
     }
   };
 
+  const onClick = (e: React.MouseEvent<HTMLElement>) => {
+    const target = e.target as HTMLElement;
```

Comment:
const onClick = ({ currentTarget }: React.MouseEvent<HTMLSpanElement>) => {
    const target = currentTarget;
    
We can get ride of a few `as` castings.